src/Component/Formbook.js

```diff
@@ -1,25 +1,20 @@
 import React, { useState } from 'react';
-import './Formbook.css';
 import { useDispatch } from 'react-redux';
-import { AddBooks } from '../redux/books/books';
-
-let counter = 3;
+import { nanoid } from '@reduxjs/toolkit';
+import { postData } from '../redux/books/bookapi';
+import './Formbook.css';
 
 const Formbook = () => {
   const [title, setTitle] = useState('');
   const [author, setAuthor] = useState('');
-  const [categories, setCategories] = useState('');
+  const [category, setCategories] = useState('');
   const dispatch = useDispatch();
   const handleclick = (e) => {
     e.preventDefault();
-    dispatch(
-      AddBooks({
-        id: counter += 1,
-        bookcategory: categories,
-        Title: title,
-        Author: author,
-      }),
-    );
+    const newbook = {
+      id: nanoid(), title, author, category,
+    };
+    dispatch(postData(newbook));
   };
   return (
     <form onSubmit={handleclick}>
```

Comment:
- [ Required ] It would be good to clear the contents of the `title input` field and `author input` field once a book has been added to the page this will give a good User Experience should someone want to add more than two books. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-01-28 13-42-03](https://user-images.githubusercontent.com/16392046/215262469-b53b67b6-3703-42ce-afcb-747dba4c5396.png)

Reply:
Fixed.